loopd/run.go

```diff
@@ -204,6 +197,13 @@ func Run(rpcCfg RPCConfig) error {
 	logWriter := build.NewRotatingLogWriter()
 	SetupLoggers(logWriter, shutdownInterceptor)
 
+	// Special show command to list supported subsystems and exit.
+	if config.DebugLevel == "show" {
```

Comment:
Is this fix effective because `if err := Validate(&config)` precedes now?

Reply:
hmm let me check

Reply:
yeah so debuglevel validation seems not part and not necessary to validate so it should be fixed